lib/guardian.ex

```diff
@@ -507,7 +527,7 @@ defmodule Guardian do
   * `:week` | `:weeks`
 
   See the documentation for your TokenModule for more information on
-  which options are available.
+  which options are available for your TokenModule.
```

Comment:
What is "TokenModule"?  It's capitalization and naming suggest it's a module that exists, but I believe that is not the case?